rotkehlchen/accounting/structures/base.py

```diff
@@ -119,6 +120,17 @@ def __init__(
         self.notes = notes
         self.identifier = identifier
 
+        # Check that the received event type and subtype is a valid combination
+        if __debug__:  # noqa: SIM102
+            if (
+                (std_mapping := EVENT_CATEGORY_MAPPINGS.get(self.event_type)) is None or
+                std_mapping.get(self.event_subtype) is None
+            ):
+                raise AssertionError(
+                    'Unexpected event type and subtype pair provided: '
```

Comment:
The code style is if any part of a multiline string is an f string add it to all

```suggestion
                    f'Unexpected event type and subtype pair provided: '
```

Reply:
Resolved with suggested change.